color.cpp

```diff
@@ -87,6 +87,13 @@ std::string to_str(const color& in_color) noexcept
   return msg;
 }
 
+bool is_first_color_winner(const color& c1, const color& c2)
+{
+    double hue1 = calc_hue(c1);

```

Comment:
This is a bit of nitpicking, yet here goes: this should be const double, as hue_1 is not changed. Still, the tests pass, so I will make that into a newbie Issue 🙂